src/Features/CSharp/Portable/Completion/CompletionProviders/XmlDocCommentCompletionProvider.cs

```diff
@@ -62,28 +62,26 @@ internal override bool IsInsertionTrigger(SourceText text, int characterPosition
                 }
             }
 
-            // User is trying to write a name, try to suggest only names.
-            if (token.Parent.IsKind(SyntaxKind.XmlNameAttribute) ||
-                (token.Parent.IsKind(SyntaxKind.IdentifierName) && token.Parent.IsParentKind(SyntaxKind.XmlNameAttribute)))
+            string elementName, attributeName;
+            if (!(trigger.Kind == CompletionTriggerKind.Insertion && trigger.Character == ' ') &&
+                IsAttributeValueContext(token, out elementName, out attributeName))
```

Comment:
Put some general comments here explaining what our trigger are and what sort of special behavior we want associated iwth each.